app/cdap/components/Replicator/Create/Content/PluginConfig/index.tsx

```diff
@@ -169,7 +176,7 @@ const PluginConfigView: React.FC<IPluginConfigProps> = ({
     return isPropertyFilled.filter((propertyValue) => !propertyValue).length > 0;
   }
 
-  function handleNext() {
+  function returnFilteredValues() {
```

Comment:
nit 1: This looks like a useEffect dependencies lint error to me. I'll leave this up to you consider moving this into a `const filterValues = useCallback(values => ..., [pluginWidget])`, then I'd `useEffect(() => ..., [filterValues, values])`, to ensure that the filterValues function is integrated into effect causality. Or even just `const filteredValues = useMemo(() => ..., [pluginWidget, values])`.

nit 2: I would personally name this function as a verb on what it does to the data, since it's doing a pure calculation. To my mental natural language parser, "return" in the function name implies that the function will cause something else outside itself to return, not merely that the rest of the name is what it's returning.

Reply:
hmm nit1 is a bit confusing - renamed to filtered values though